embedded/projects/SH-Xpresso-LPC54102/sources/app/sensoracq_t.c

```diff
@@ -129,7 +130,7 @@ static void SensorDataHandler(InputSensor_t sensorId, uint32_t timeStamp)
         if ((sMagDecimateCount++ % MAG_DECIMATE_FACTOR) == 0 )
         {
             /* Read mag Data - reading would clear interrupt also */
-            Mag_ReadData( &magData );
+            Driver_Mag.ReadData(&magData,0);
```

Comment:
What does it mean to pass 0 for `num_samples` ? Does it mean read ALL the samples?  Or is the intent here to read 1 sample?